dev-tools/cloud/docker/build.sh

```diff
@@ -9,9 +9,9 @@ set -eu
 REPO_ROOT=$(cd $(dirname $(readlink -f "$0"))/../../.. && pwd)
 
 USER_NAME=${USER}
-CI_ELASTIC_AGENT_DOCKER_IMAGE=docker.elastic.co/observability-ci/elastic-agent
+CI_ELASTIC_AGENT_DOCKER_IMAGE=${CI_ELASTIC_AGENT_DOCKER_IMAGE:-"docker.elastic.co/observability-ci/elastic-agent"}
 
-DEFAULT_IMAGE_TAG=8.8.0-94522507-SNAPSHOT
+DEFAULT_IMAGE_TAG=8.9.0-5c23b5b2-SNAPSHOT
```

Comment:
Perhaps you should source `dev-tools/integration/.env` and use the version from there?

Reply:
@nchaulet - This still needs to be done, correct? Agree w/ Michel that we should use the `.env` file's configured version if possible here.

Reply:
Yes I am going to address that